components/service/glean/docs/pings/metrics.md

```diff
@@ -10,17 +10,22 @@ length of this window.
 
 ## Scheduling
 The desired behaviour is to collect the ping at the first available opportunity after 4AM local
-time on a new calendar day. This breaks down into two scenarios:
+time on a new calendar day. This breaks down into three scenarios:
 
-1. the application was just started (after a crash or a long inactivity period);
-2. the application was open and the 4AM due time was hit.
+1. the application was just installed;
+2. the application was just started (after a crash or a long inactivity period);
+3. the application was open and the 4AM due time was hit;
 
-In the first case, if the `metrics` ping was already collected on the current calendar day, a new
+In the first case, since the application was just installed, if the due time for the current calendar
```

Comment:
At first I thought this was a little weird to send a metrics ping immediately on a fresh install since there would likely be nothing to send, but I realized that it helps to fill in the missing metrics pings we saw from the initial analysis and some metrics do get set at startup.